app/uk/gov/hmrc/tai/connectors/IabdConnector.scala

```diff
@@ -39,11 +40,15 @@ class IabdConnector @Inject()(
     if (taxYear > TaxYear()) {
       Future.successful(Json.arr())
     } else if (featureTogglesConfig.desEnabled) {
-      val hcWithHodHeaders = hc.withExtraHeaders("Gov-Uk-Originator-Id" -> desConfig.originatorId)
+      val hcWithHodHeaders = hc.withExtraHeaders(
+        "Gov-Uk-Originator-Id" -> desConfig.originatorId,
+        "CorrelationId"        -> UUID.randomUUID().toString)
```

Comment:
What about the session and request IDs?

Reply:
Session and request Ids are coming with incoming hc itself . Here we are adding extra headers for orgininator and correlation id